src/Keboola/DbExtractor/RetryProxy.php

```diff
@@ -86,7 +86,8 @@ public function call(callable $action, array $arguments = [])
             if ($this->retryPolicy->canRetry($retryContext)) {
```

Comment:
we should move the %s. Retrying... [%dx]', message instead of backing off, not it gives `Exporting to in.c-main.sales
Dead connection: PDO::query(): MySQL server has gone away. Retrying... [1x]` on 0 retries

Reply:
good point